pages/governance.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: otherpage
+title: Governance
+---
+
+Project leadership is provided by a group of contributors, known as the
+*MDAnalysis Core Developers* ([@MDAnalysis/coredevs][]) who have produced substantial contributions over an extended period and who actively review issues and participate in discussions on GitHub Discussions and Discord. 
+
+## MDAnalysis Core Developers
+
+The **Core Developers** lead the MDAnalysis project and are responsible to
+the community and to [NumFOCUS][], our fiscal sponsor. They **represent
+the project publicly** and **vote to make decisions for the project**.
+
+PIs on a grant submitted by MDAnalysis via NumFOCUS must be Core Developers
```

Comment:
```suggestion
Principal Investigators (PIs) on a grant submitted by MDAnalysis via NumFOCUS must be Core Developers,
```

Reply:
@orbeckst  is this just 'via numfocus' or in general?

Reply:
We said always. You can't promise work "on MDAnalysis" if you are not in a position to speak for the project.

Reply:
I now realize that the [original about page](https://www.mdanalysis.org/about/#mdanalysis-core-developers) does say

> PIs on a grant submitted by MDAnalysis via NumFOCUS must be Core Developers while co-PIs do not have to be Core Developers.

Thus, for the purpose of restructuring the web site **we should keep the old sentence intact.**

(Changing it is a different matter.)